astrobin/templates/base/user_links_entries.html

```diff
@@ -20,7 +20,14 @@
 <li>
     <a href="{% url 'user_page' request.user.username %}">
         <i class="icon-picture"></i>
-        {% trans "Gallery" %}
+        {% trans "Public Gallery" %}
+    </a>
+</li>
+
+<li>
+    <a href="{% url 'user_page' request.user.username %}?staging">
+        <i class="icon-lock"></i>
+        {% trans "Staging Gallery" %}
```

Comment:
```suggestion
        {% trans "Staging Area" %}
```

It would be better to keep the "Staging Area" convention. It's still named that way in a few other places and the translations are already in place.